lib/server.rb

```diff
@@ -38,28 +38,28 @@ def load_instance_variables(init_opts)
     @server_port = init_opts[:server_port]
     @connection_timeout = init_opts[:connection_timeout]
     @outp_dir = init_opts[:outp_dir]
-    @l_script_file = init_opts[:l_script_file]
     @r_script_file = init_opts[:r_script_file]
   end
 
   def check_script_file
     logger('debug', 'server/initialize') { 'checking script file on remote' }
-    if !@l_script_file.nil? then deploy_script_file
-    elsif !@winrm_fs.exists?(@r_script_file)
+    deploy_script_file
+    unless @winrm_fs.exists?(@r_script_file)
       raise ServerError.new('server/initialize'),
             'toolsHCK.ps1 script was not found on remote.'
     end
     logger('debug', 'server/initialize') { 'checked' }
   end
 
   def deploy_script_file
+    tools_hck = File.expand_path('../tools/toolsHCK.ps1', __dir__)
     logger('debug', 'server/initialize') { 'deploying script file on remote' }
-    unless File.file?(@l_script_file)
+    unless File.file?(tools_hck)
       raise ServerError.new('server/initialize'),
-            "can't find the l_script_file specified."
+            "can't find the file specified."
     end
```

Comment:
Please remove this. We don't have to check the existence of files in general (so you don't have to check if a file exists when `require`, for example).